nbs/api/14_quarto.ipynb

```diff
@@ -50,7 +50,8 @@
     "from fastcore.shutil import rmtree,move,copytree\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        for dabs, name in files:
</pre><p>slight rewrite, no need for <code>if</code> statement: </p><p><code>    for dabs, name in files:</code></p><p><code>        drel = dabs.relative_to(path)</code></p><p><code>        _dir = dir_struct</code></p><p><code>        for subdir in drel.parts:</code></p><p><code>            _dir = _dir.setdefault(subdir, dict())</code></p><p><code>        _dir[name] = name</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Looks good to me. Thanks.</p>